src/pages/sns.tsx

```diff
@@ -0,0 +1,480 @@
+                md:absolute 
+                md:w-[640px] md:max-w-none md:left-auto md:translate-x-0 md:right-[-50px] md:top-[-30px]"
+              src="/img/sns/sns.webp"
+            ></img>
+            <div className="md:mx-auto md:w-10/12 relative">
+              <motion.h2
+                className="tw-heading-4 md:tw-heading-3 md:w-5/10 mb-6 md:my-8"
+                variants={transitions.item}
+              >
+                Own and shape the services and applications you love.
+              </motion.h2>
+
+              <motion.p variants={transitions.item} className="tw-lead-sm mb-3">
+                Announcment soon on DSCVR.
+              </motion.p>
+              <motion.p variants={transitions.item}>
+                <Link
+                  className="link-primary-light"
+                  href="https://dscvr.one/p/internet-computer"
+                >
+                  Join the Internet Computer Portal
+                  <ExternalLinkIcon className="inline-block align-bottom ml-2"></ExternalLinkIcon>
+                </Link>
+              </motion.p>
+            </div>
+            <motion.img
+              src={BlobWhite}
+              className="absolute pointer-events-none max-w-none w-[800px] right-[-250px] top-[-150px] md:w-[1500px]  md:right-[-550px] translate-x-[200px] md:top-[-400px]"
+              alt=""
+            />
+          </AnimateSpawn>
+        </section>
+        <AnimateSpawn
+          className="container-10 py-24 md:py-40 relative"
+          variants={transitions.container}
+        >
+          <div className="">
+            <AnimateSpawn
+              el={motion.img}
+              variants={transitions.item}
+              src={BlobBlue}
+              className="absolute pointer-events-none
+              max-w-none
+              w-[1000px]
+              bottom-[-400px]
+              right-[-600px]
+              sm:bottom-[-600px]
+              sm:w-[1200px]
+              sm:right-[-500px]
+            "
+            />
+          </div>
+          <div className="flex flex-col sm:flex-row gap-8 sm:gap-1/10">
+            <motion.div variants={transitions.item} className="flex-1">
+              <motion.h2
+                className="tw-heading-5 md:tw-heading-3 mb-10 md:mb-16"
+                variants={transitions.item}
+              >
+                What an SNS DAO is <span className="text-razzmatazz">NOT</span>
+              </motion.h2>
+              <div className="md:w-8/10">
+                <h3 className="tw-heading-6 md:tw-heading-5 mb-4">
+                  Not an ICO
+                </h3>
+                <p className="tw-paragraph mb-10">
+                  No funds are handed over to developers. The creation of an SNS
+                  is initiated by a decentralized entity (
+                  <Link
+                    className="text-infinite underline hover:no-underline hover:text-black"
+                    href="https://nns.ic0.app/"
```

Comment:
I think we mix here different meanings of "NNS".
In the sentence we refer to the NNS as a DAO (i.e., the DAO that runs the IC).
However, we link to _one_ frontend that allows to interact with the NNS. 
I propose we rather link to an explanation of what the NNS is.